temoa/model_checking/element_checker.py

```
@@ -33,102 +8,151 @@ class to hold members of "validation sets" used by loader to validate elements a
+        """
+        Sets or updates the validation exceptions.
+
+        Args:
+            exception_loc: The index for the exception.
+            exception_vals: An iterable of regex patterns.
+
+        Returns:
+            The instance (self) to allow for method chaining.
+        """
+        if not (isinstance(exception_loc, int) and exception_loc >= 0):
+            raise ValueError('exception_loc must be a non-negative integer')
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)